openapi/nexus.json

```diff
@@ -9727,7 +9816,7 @@
           "last_address"
         ]
       },
-      "AddressLotBlockCreate": {
+      "AddressLotBlock2": {
```

Comment:
just looking through this quickly, but I think we should avoid adding **more** of these name conflicting types that result in the `2` suffix.